tinode-db/main.go

```diff
@@ -218,18 +218,27 @@ func main() {
 		log.Println("Database reset requested")
 	} else {
 		log.Println("Database exists, DB version is correct. All done.")
-		return
+		os.Exit(0)
```

Comment:
Also add `os.Exit(0)` as the very last line of this file, after `genDb(&data)`. Github does not let me add a comment there.

Reply:
Done.